src/pages/waypoint/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -7,7 +7,7 @@ import {
   TutorialPagePaths,
```

Comment:
What prompted these changes? No issues, purely curious. 😊

Reply:
I think I got some warning or issue re: destructuring on an import!

Reply:
Ah okay, that makes sense!